synfig-studio/src/gui/preview.cpp

```diff
@@ -476,21 +476,21 @@ Widget_Preview::Widget_Preview():
 	//halt render
 	button = manage(new Gtk::Button(/*_("Halt Render")*/));
 	button->signal_clicked().connect(sigc::mem_fun(*this, &Widget_Preview::stoprender));
-	IMAGIFY_BUTTON(button,Gtk::Stock::STOP, _("Halt render"));
+	IMAGIFY_BUTTON(button,Gtk::Stock::STOP, _("Stop rendering"));
 
 	toolbar->pack_start(*button, Gtk::PACK_SHRINK, 0);
 
 	//re-preview
 	button = manage(new Gtk::Button(/*_("Re-Preview")*/));
 	button->signal_clicked().connect(sigc::mem_fun(*this, &Widget_Preview::repreview));
-	IMAGIFY_BUTTON(button, Gtk::Stock::EDIT, _("Re-preview"));
+	IMAGIFY_BUTTON(button, Gtk::Stock::EDIT, _("Reopen the Preview Settings Dialog"));
```

Comment:
Maybe "Edit (or change) Preview Settings"?

Reply:
What about `Open Preview Settings`?

Reply:
I've changed the tooltip to `Open Preview Settings Dialog` and the pencil icon in the Preview window:
![preview-open-change-icon](https://user-images.githubusercontent.com/6705690/103139871-278bee80-46e1-11eb-8296-ff3ed1969c5c.png)

This way it's consistent with current toolbar where I've also matched the tooltips for Render and Preview dialogs.